chef-utils/lib/chef-utils/internal.rb

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright:: Copyright 2018-2019, Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module ChefUtils
+  module Internal
+    extend self
+
+    #
+    # This contains a lot of glue code to automagically wire up Cheffy things from chef-client.
```

Comment:
if you stick this commend above the internal module yard will do the right thing with the display